### ISSUE TITLE
scan remote git repository

### ISSUE THREAD

**Issue by markuman** [Skyscanner/whispers#60]
it would be nice if whisper can scan remote git repositories.

**Comment by adeptex** [comment]
hey @markuman, thanks for posting. 

For whispers to scan a git repo it would first have to locally clone the repo, and then run as per usual. I think it would be simpler to leave the cloning part outside of the tool itself, because there may be other version control systems (ie: SVN, TFS, etc) that may be used instead of git. Provisioning of the source code will be difficult to implement at code level.

On the other hand, implementing this in a script would be much simpler, for example:
```bash
git clone https://github.com/org/repo /tmp/local_copy
whispers -c config.yml /tmp/local_copy
```

**Comment by adeptex** [comment]
Hey @markuman, i will close this one for the time being. Please feel free to reopen if you have any further comments on this. 

The other issues you submitted are now available in 1.5.0 release :] thanks again for feedback!

Cheers
